test/deploy.sh

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env bash
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+###############################################################################
+
+set -e
+set -u
+set -o pipefail
+
+ROOT="${DIR}/.."
+source "${ROOT}/test/common.sh"
+
+function cleanup {
```

Comment:
Maybe, going to punt for now as I have enough of these bulky retry loops in here. If I add a better `retry` helper function to common, I could use it here as well and will do so.